pkgs/development/tools/parsing/antlr/3.5.nix

```diff
@@ -1,18 +1,23 @@
-{stdenv, fetchurl, jre}:
+{stdenv, fetchurl, fetchFromGitHub, jre}:
 
 stdenv.mkDerivation rec {
-  name = "antlr-${version}";
+  pname = "antlr";
   version = "3.5.2";
-  src = fetchurl {
-    url ="https://www.antlr3.org/download/antlr-${version}-complete.jar";
+  jar = fetchurl {
```

Comment:
What's up with these antlr changes?

Reply:
This includes the antlr libraries in the package output, which are not currently present in `antlr 3.5.nix`

My initial approach was to add a new package `libantlr3cpp` that exposes the libs only, and @aanderse [suggested to modify the existing package](https://github.com/NixOS/nixpkgs/pull/61438#discussion_r306251757) instead!